### DISCUSSION TITLE
Reviewing duplicates is in random order

### DISCUSSION THREAD

**Discussion by johanjongsma** [immich-app/immich#10424] [Feature Request]
### The bug

While reviewing the (many) duplicates, they appear in random order. Would be better to show them in alphabetical order of the original or in order of timestamp.

### The OS that Immich Server is running on

Docker

### Version of Immich Server

v1.106.4

### Version of Immich Mobile App

v1.106

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
not relevant
```

### Your .env content

```Shell
not relevant
```

### Reproduction steps

```bash
1. go to duplicates
2. process the first
3. process the second, which is totally unrelated to the previous
```

**Comment by starbuck93** [comment]
I have about 5000 duplicate groups, I think organizing them by date would help me speed up cleaning them up
